### PR TITLE
Issue 256

### LINKED ISSUE #256: Verify page-specific CSS is prefixed
**Please describe what has to be done**
Most CSS styles are in the main.scss file. Some pages have individual CSS files next to them. Because of how React works, these all get combined and can potentially overwrite each other even though they appear to be separate. What needs to happen is make sure that each individual CSS file uses prefixes so that there is no conflicts.

For consistency, let's do it like `.FolderName-item`.

**Additional context**
Example: 

PollResults/
PollResults/PollResults.js
PollResults/PollResults.scss -> 
```
.PollResults-graph{
  width: 60%;
}
.PollResults-questions{
  margin-right: 50px;
  width: calc(25% - 50px);
}
```

### PR DESCRIPTION
**Pull request by aleong1**
### Description of change(s), including why:

Went through all the individual CSS files and made sure they all had the same prefix: .Foldername-item and updated the corresponding js files to match

### Type of change(s)

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Any work to still do or other concerns:

- (none)

### Closing issues:

- Closes #256

### CODE REVIEW

**Review by annieshi6 — COMMENTED** [review]
For the most part, everything looks good! The only file I would suggest that you probably change back is index.css bc body and code are not classes.

Also note: most pull requests you're going to want to merge into frontend or backend, instead of directly into master but for this pr you're probably fine since it shouldn't break anything.

Open question to anyone else that looks at this pr: is there a purpose for index.css? if not, can we safely throw it out?

**Review by annieshi6 — DISMISSED** [review]

File: PollBuddy-Server/frontend/src/pages/PollViewer/PollViewer.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default class PollViewer extends Component {
     return (
       <MDBContainer>
         <MDBContainer className="page">
-          <MDBContainer className="box pollview_answers">
+          <MDBContainer className="box Pollview-answers">
```
Comment (hbh7) [inline PollBuddy-Server/frontend/src/pages/PollViewer/PollViewer.js:14]: `Pollview` -> `PollViewer` and the ones below with capitalization too

File: PollBuddy-Server/frontend/src/pages/Homepage/Homepage.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
-.homepage_logo {
+.Homepage-logo {
```
Comment (hbh7) [inline PollBuddy-Server/frontend/src/pages/Homepage/Homepage.scss:1]: Let's stick with Homepage rather than Home

**Review by hbh7 — CHANGES_REQUESTED** [review]
Just so things look a bit nicer, can you also swap out the `_`'s for `-`'s? 
Ex: `Footer-foot_links` -> `Footer-foot-links`

There's also a few naming consistency issues (looks like they were that way before) that would be great if you could fix up too. I've left comments in the code. 

Otherwise though looks great!

**Review by annieshi6 — DISMISSED** [review]

**Review by hbh7 — APPROVED** [review]
Looks perfect! Nicely done

**Comment by hbh7** [comment]
Oh actually just one last thing, merge into frontend instead. I'll make the change and it'll auto dismiss the reviews but that's fine, I saw them so I'll merge anyways.

**Pull request merged**